src/openrct2/interface/Theme.cpp

```diff
@@ -173,6 +173,7 @@ static const WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_NETWORK_STATUS),                 STR_THEMES_WINDOW_NETWORK_STATUS,                 COLOURS_1(COLOUR_LIGHT_BLUE                                                                                                    ) },
     { THEME_WC(WC_SERVER_LIST),                    STR_SERVER_LIST,                                  COLOURS_2(COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                                                   ) },
     { THEME_WC(WC_CHAT),                           STR_CHAT,                                         COLOURS_1(TRANSLUCENT(COLOUR_GREY)                                                                                             ) },
+    { THEME_WC(WC_CONSOLE),                        STR_CONSOLE,                                      COLOURS_1(TRANSLUCENT(COLOUR_BLACK)                                                                                            ) },
```

Comment:
I thought it was translucent blue, not black?

Reply:
It was, but I thought I'd stick with something a bit more neutral as the default. Opinions on it were varied enough as it is.

Reply:
Please change it back. If opinions on it are varied enough, then changing defaults is not a very prudent thing.

Reply:
Okay, I have changed it back to light blue.

My personal preferences aside, there's a caveat, though: currently, the theme engine does not allow us to get to `PALETTE_TRANSLUCENT_LIGHT_BLUE_HIGHLIGHT` exactly — just to `PALETTE_TRANSLUCENT_LIGHT_BLUE`. As people can now change the colour to whatever they like, this probably does not pose much of a problem, however.